clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/stats/BasicConsumerStats.java

```diff
@@ -0,0 +1,20 @@
+package com.linkedin.davinci.consumer.stats;
+
+import com.linkedin.venice.stats.AbstractVeniceStats;
+import io.tehuti.metrics.MetricsRepository;
+import io.tehuti.metrics.Sensor;
+import io.tehuti.metrics.stats.Max;
+
+
+public class BasicConsumerStats extends AbstractVeniceStats {
```

Comment:
Besides lag, should we also have consumption rate and record count for debugging purpose in case lag is high/stuck?

Reply:
That's an interesting idea, but not sure how best to do it.  record count/rate isn't really controlled by us.  It's really an artifact of how often the user calls poll().  I'm concerned that stats that show this is small will be distracting, and the partner will insist it's on us.  Though I think it'd be useful in so far as the partner wants to just have stats for the performance of their consumer.